test/system/helpers.bash

```diff
@@ -8,7 +8,7 @@ QUADLET=${QUADLET:-quadlet}
 PODMAN_TEST_IMAGE_REGISTRY=${PODMAN_TEST_IMAGE_REGISTRY:-"quay.io"}
 PODMAN_TEST_IMAGE_USER=${PODMAN_TEST_IMAGE_USER:-"libpod"}
 PODMAN_TEST_IMAGE_NAME=${PODMAN_TEST_IMAGE_NAME:-"testimage"}
-PODMAN_TEST_IMAGE_TAG=${PODMAN_TEST_IMAGE_TAG:-"20221018"}
+PODMAN_TEST_IMAGE_TAG=${PODMAN_TEST_IMAGE_TAG:-"20240123"}
```

Comment:
This is the cause of the system-test failures. Why are you bumping this? If there is no compelling reason, please just revert.

Reply:
@cevich commented at https://github.com/containers/podman/pull/20906#issuecomment-2000083744 . Did I bump in the wrong place?

Reply:
I see the comment but not what it refers to. Is there any way to get more context? If not or if it's too hard, I suggest simply reverting that change, i.e. going back to `20221018`